site/en/guides/border-router/mdns-discovery.md

```diff
@@ -0,0 +1,123 @@
+# mDNS Discovery
+
+OTBR registers a `_meshcop._udp` service named `OpenThread_BorderRouter`. This
+service is configured in the [OTBR CMakeLists.txt file](https://github.com/openthread/ot-br-posix/blob/main/CMakeLists.txt#L36).
+
+There are several ways to discover your Thread network.
+
+## DNS Service Discovery (DNS-SD)
+
+Use `dns-sd` to browse for a service on `raspberrypi.local.:49155`:
+
+```
+$ dns-sd -L "OpenThread_BorderRouter" _meshcop._udp local
+```
+
+<pre>
+Lookup OpenThread_BorderRouter._meshcop._udp.local
+DATE: ---Tue 16 Nov 2021---
+13:31:03.197  ...STARTING...
+13:31:03.350  OpenThread_BorderRouter._meshcop._udp.local. can be reached at raspberrypi.local.:49155 (interface 3)
+</pre>
+
+Get the IP addresses associated with your OTBR domain name:
+
+```
+$ dns-sd -G v4/v6 {raspberrypi.local}
+```
+
+<pre>
+DATE: ---Tue 16 Nov 2021---
+14:21:29.485  ...STARTING...
+Timestamp     A/R Flags if Hostname           Address                                      TTL
+14:21:29.486  Add     3  3 raspberrypi.local. FDDE:AD11:11DE:0000:74D0:6FC9:6BE6:3582%&lt;0&gt;  120
+14:21:29.486  Add     3  3 raspberrypi.local. FD00:0000:0000:0000:74D0:6FC9:6BE6:3582%&lt;0&gt;  120
+14:21:29.486  Add     3  3 raspberrypi.local. FE80:0000:0000:0000:74D0:6FC9:6BE6:3582%eth0 120
+14:21:29.486  Add     3  3 raspberrypi.local. FE80:0000:0000:0000:287F:87CA:F4B3:498A%eth0 120
+14:21:29.486  Add     2  3 raspberrypi.local. 192.168.0.10                                 120
+</pre>
+
+Ping your IP address. From the `dns-sd` results, choose an address that's
+reachable from your network, for example the global `<0>` address
+**FD00:0000:0000:0000:74D0:6FC9:6BE6:3582**:
```

Comment:
Can you compress this to FD00::74D0:6FC9:6BE6:3582 and put it in code font?

Reply:
We should be able to used compressed form.

Reply:
np - done.